about.md

```diff
@@ -12,9 +12,9 @@ scribble: scribble6
 
 So what is Rails Girls Summer of Code all about?
 
-Rails Girls Summer of Code is a global fellowship program aimed at bringing
+Rails Girls Summer of Code is an [award-winning](https://rubyheroes.com/heroes/2016){:target="_blank"} global fellowship program aimed at bringing
```

Comment:
I'd rather link to the blog post because it explains a bit more? https://railsgirlssummerofcode.org/blog/2016-06-01-ruby-heroes-2016 or do you like the ruby heroes page better?